src/ProcessBase.php

```diff
@@ -172,6 +172,8 @@ public function getOutputAsJson()
         }
         $output = preg_replace('#^[^{]*#', '', $output);
         $output = preg_replace('#[^}]*$#', '', $output);
+        $output = str_replace('\\"', '"', $output);
+        $output = preg_replace('#\\\\{2}#', '\\', $output);
```

Comment:
Could you explain what this is for, e.g. with a comment?

Reply:
Comments added.